lib/datatables_crud/controller.rb

```diff
@@ -2,7 +2,7 @@ module DatatablesCRUD
   module Controller
     def crud_actions(actions)
       actions = [:index, :show, :new, :create, :edit, :update, :destroy] if actions == :all
-      before_filter :load_resource, :only => [:show, :edit, :update, :destroy].select { |action| actions.include?(action) }
+      before_action :load_resource, :only => [:show, :edit, :update, :destroy].select { |action| actions.include?(action) }
```

Comment:
will this changes also be merged to master in another PR?

Reply:
If I understand it correctly, then in reaktor - yes, but as they are not used it won't cause any problems: https://github.com/konvenit/reaktor/pull/918

In miceplace - not yet. There are many changes in the branch, which miceplace is using, so it will require a separate ticket to find out how to nicely merge these changes into master, so that they won't break reaktor. It may be easier than it looks, because it's possible that many functions are used always by only one of these apps.